components/resources/wallet_strings.grdp

```diff
@@ -99,6 +99,11 @@
   <message name="IDS_BRAVE_WALLET_CHOOSE_ASSET" desc="Send tab choose asset button.">Choose Asset</message>
   <message name="IDS_BRAVE_WALLET_ORDINALS_WARNING_MESSAGE" desc="Warning message when sending BTC.">Brave Wallet does not currently support Bitcoin NFTs (ordinals). Sending BTC from an address that has ordinals may result in its ordinals being transferred inadvertently.</message>
   <message name="IDS_BRAVE_WALLET_USER_UNDERSTANDS_LABEL" desc="Message for use to acknowledge a decision">I understand</message>
+  <message name="IDS_BRAVE_WALLET_CHOOSE_RECIPIENT" desc="Select address modal title">Choose recipient</message>
+  <message name="IDS_BRAVE_WALLET_MY_ADDRESSES" desc="Select address modal my addresses label">My addresses</message>
+  <message name="IDS_BRAVE_WALLET_ADDRESS_OR_DOMAIN_PLACEHOLDER" desc="Seach address or domain input placeholder">Enter public address or ENS domain.</message>
```

Comment:
Why only ENS?
We also have SNS and Unstoppable Domains

Reply:
Updated to `Enter public address or domain.`